src/main/eo/org/eolang/math/number.eo

```diff
@@ -55,17 +55,38 @@
   [] > is-float
     ^.n > value!
     if. > @
-      value.eq 0.0
+      ^.is-float-zero
       TRUE
       if.
-        value.eq 0
+        ^.is-int-zero
         FALSE
         eq.
           div.
             value
             value
           1.0
 
+  # Checking if number is integer 0
```

Comment:
@c71n93 [suggestion]: maybe move these methods inside `is-float` and `is-int` objects? I think there's no need to have them in `number` (at least for now)